### PR TITLE
fixed infinite loop and contion change

### PR DESCRIPTION
**Pull request by jeelion22**
fixed infinite loop execution when the same letter is entered repeatedly, while loop condition is changed to True. simplified the statements for the inputs of all case letters.

### CODE REVIEW

**Review by josef-o — APPROVED** [review]
The changes are good.

One last thing I'd like to comment on is that on line `66` when you catch any Exception you then print out the traceback which is not user friendly.

Maybe just printing `"Something went wrong, please try again."` looks better.

But that is just me nitpicking, the code is fine and you can merge it if you'd like.

**Review by rzmobiledev — COMMENTED** [review]
I am pretty sure that you only want to check guess word inside stored_letters because used_letters at first time is still empty list

**Comment by jeelion22** [comment]
thank you so much for the suggestion and are welcome ***@***.****

**Review by rzmobiledev — APPROVED** [review]

File: hangman.py [hunk 1]
```diff
@@ -63,7 +63,7 @@
             break
 
     except Exception as e:
-        print(str(e))
+        print("Something went wrong, please try again.")
```
Comment (josef-o) [inline hangman.py:66]: Thanks for accepting my suggestion, appreciate it!

**Pull request merged**